Upload without mapping file (debug) fails with version 1.2

When uploading a build type to that does not produce a mapping the plugin will crash whenever `uploadMappingFiles` is enabled, this bug was introduced here: 

https://github.com/oliviergauthier/gradle-appcenter-plugin/blame/02f6f19131d576d4fd960af4e00d4f3511c17030/src/main/kotlin/com/betomorrow/gradle/appcenter/AppCenterPlugin.kt#L74

And can be reproduced by running (with: `uploadMappingFiles` set to true)
`gradlew app:appCenterUploadDebug --stacktrace`

Stacktrace:

```
Caused by: java.lang.IllegalStateException: Expected file collection to contain exactly one file, however, it contains no files.
        at org.gradle.api.internal.file.AbstractFileCollection.getSingleFile(AbstractFileCollection.java:86)
        at com.betomorrow.gradle.appcenter.AppCenterPlugin$handleVariant$$inlined$let$lambda$1$1$2.invoke(AppCenterPlugin.kt:74)
        at com.betomorrow.gradle.appcenter.AppCenterPlugin$handleVariant$$inlined$let$lambda$1$1$2.invoke(AppCenterPlugin.kt:12)
        at com.betomorrow.gradle.appcenter.tasks.UploadAppCenterTask.upload(UploadAppCenterTask.kt:40)
```

--- a/src/main/kotlin/com/betomorrow/gradle/appcenter/AppCenterPlugin.kt
+++ b/src/main/kotlin/com/betomorrow/gradle/appcenter/AppCenterPlugin.kt
@@ -71,7 +71,7 @@ class AppCenterPlugin : Plugin<Project> {
                         uploadTask.notifyTesters = appCenterApp.notifyTesters
 
                         if (appCenterApp.uploadMappingFiles) {
-                            uploadTask.mappingFileProvider = { variant.mappingFileProvider.get().singleFile }
+                            uploadTask.mappingFileProvider = { variant.mappingFileProvider.get().firstOrNull() }
                         } else {
                             uploadTask.mappingFileProvider = { null }
                         }

